etc/webapi.xml

```diff
@@ -58,6 +58,13 @@
         </resources>
     </route>
 
+    <route url="/V1/adyen/me/stateData" method="POST">
```

Comment:
what is the `me` in the url?

Reply:
I was experimenting with this. If "me" is used in the path then some params can be forced by M2, like customerId. But for now we don't need it
```suggestion
    <route url="/V1/adyen/stateData" method="POST">
```